--- ol/renderer/canvas/tilelayer.js	2017-03-02 13:30:25.257547216 +1000
+++ ol/renderer/canvas/tilelayer_fixed.js	2017-03-02 13:34:08.404537831 +1000
@@ -180,7 +180,7 @@
 
     var tilePixelSize = tileSource.getTilePixelSize(z, pixelRatio, projection);
     var width = Math.round(tileRange.getWidth() * tilePixelSize[0] / oversampling);
-    var height = Math.round(tileRange.getHeight() * tilePixelSize[0] / oversampling);
+    var height = Math.round(tileRange.getHeight() * tilePixelSize[1] / oversampling);
     var context = this.context;
     var canvas = context.canvas;
     var opaque = tileSource.getOpaque(projection);
